terraform/variables.tf

```diff
@@ -42,5 +42,25 @@ variable "num_nodes" {
 variable "qcow2_image" {
   type        = string
   description = "Ubuntu image for VMs - only needed for libvirt provider"
-  default     = "/bionic-server-cloudimg-amd64.img"
+  default     = "/ubuntu-18.04-server-cloudimg-amd64.img"
 }
+
+variable "overlay_cidr" {
+  type        = string
+  description = "CIDR, classless inter-domain routing"
+  #default     = "192.168.122.0/16"
```

Comment:
can we have the default here? The previous default worked fine for me. IMHO the fewer settings the user is required to tune the better.

Reply:
Done https://github.com/openebs/Mayastor/pull/268/commits/5b38dc66d2408d7b84cd517462429a5cd21232c9